openmrs/i18n/home/locale_en.json

```diff
@@ -11,5 +11,35 @@
     "MODULE_LABEL_PROGRAMS_KEY": "Programs",
     "MODULE_LABEL_IMPLEMENTER_INTERFACE_KEY": "Implementer Interface",
     "MODULE_LABEL_ATOMFEED_CONSOLE_KEY": "AtomFeed Console",
-    "MODULE_LABEL_APPOINTMENT_SCHEDULING_KEY": "Appointment Scheduling"
+    "MODULE_LABEL_APPOINTMENT_SCHEDULING_KEY": "Appointment Scheduling",
+    "HOME_LOCATION_HOSPITAL":  "Hopital de la Cité Verte",
```

Comment:
@prashantsable1991 I would suggest removing the prefix "HOME_" since the keys will be used for other modules as well (i.e. clinical and program)

Reply:
I did all suggested changes please review.